src/utils/__tests__/legacyLayerParamUtils.spec.js

```diff
@@ -282,5 +283,32 @@ describe('Test parsing of legacy URL param into new params', () => {
                 })
             })
         })
+        describe('ensure layers parameter handler for feature preselection works as intended', () => {
+            it('checks that all possible layers given as parameter return what we expect', () => {
+                const layersParams = [
+                    'layer.id;layer.id2',
+                    'layer.id,,0.3;layer.id2',
+                    'layer.id@time=1234;layer.id2',
+                    'layer.id@features=3:4:5;layer.id2',
+                    'layer.id@features=3:4:5@time=1234,f,0.2;layer.id2',
+                ]
+                const expectedResults = [
+                    'layer.id@features=1:2:3;layer.id2',
+                    'layer.id@features=1:2:3,,0.3;layer.id2',
+                    'layer.id@time=1234@features=1:2:3;layer.id2',
+                    'layer.id@features=1:2:3:4:5;layer.id2',
+                    'layer.id@features=1:2:3:4:5@time=1234,f,0.2;layer.id2',
+                ]
+                const results = []
+                layersParams.forEach((params) => {
+                    results.push(
+                        createLayersParamForFeaturePreselection('layer.id', '1,2,3', params)
+                    )
+                })
+                for (let i = 0; i < results.length; i++) {
+                    expect(expectedResults[i]).to.eq(results[i])
+                }
+            })
```

Comment:
we are not limited (or limiting ourselves) with `.it` calls while unit testing, i would prefer each of this case your are testing here would be its own `.it` with a description of what you are testing as input message

And the beauty of unit testing is that you can safely break stuff, so please also test some "unwanted" inputs to see how your code would react (users will do it for your if you don't do it here :wink: )